### DISCUSSION TITLE
Sunbeams in the cockpit

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5664] [General]
HEllo dev team,

don´t know if somebody else did a report about a possible bug in the cockpit due to sunbeams on the wrong position...

![grafik](https://user-images.githubusercontent.com/76531893/129556813-80da056c-46dc-4867-9b12-2b03d5e2cedf.png)

greets Klaus

**Comment by tracernz** [comment]
Known issue with the Asobo 3d model. Should be addressed by our own flight deck model that is in progress.
